### PR TITLE
test(node/fs): enable `test-fs-rm.js`

### PR DESCRIPTION
**Pull request by F3n67u**
this is a part of #1564

### CODE REVIEW

File: node/internal/url.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+export function toPathIfFileURL(fileURLOrPath) {
+  if (!isURLInstance(fileURLOrPath)) {
+    return fileURLOrPath;
+  }
+  return fileURLToPath(fileURLOrPath);
+}
+
+export default {
+  toPathIfFileURL,
```
Comment (wafuwafu13) [inline node/internal/url.js:110]: Maybe you can use here.
https://github.com/denoland/deno_std/blob/main/node/url.ts

I've implemented here with reference to https://github.com/nodejs/node/blob/master/lib/internal/url.js

Comment (F3n67u) [reply node/internal/url.js]: thanks for the review. I removed the `url.js` file and use `deno_std/node/url.ts` instead.

File: node/url.ts [hunk 2]
```diff
@@ -325,6 +332,7 @@ export default {
   format,
   fileURLToPath,
   pathToFileURL,
+  toPathIfFileURL,
```
Comment (kt3k) [inline node/url.ts:335]: Is this used somewhere?

File: node/_tools/config.json [hunk 3]
```diff
@@ -129,6 +129,7 @@
       "test-event-emitter-symbols.js",
       "test-events-list.js",
       "test-events-once.js",
+      "test-fs-rm.js",
```
Comment (kt3k) [inline node/_tools/config.json:132]: nice 👍

**Review by kt3k — COMMENTED** [review]
Mostly looks good. Left one question

File: node/url.ts (hunk 2 above)
Comment (F3n67u) [reply node/url.ts:335]: it's imported in `node/internal/fs/utils.js`

**Review by kt3k — APPROVED** [review]
@F3n67u LGTM. Thank you for your contribution!

**Pull request merged**